templates/selected_course/selected_course.html

```diff
@@ -4,34 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{{ course.course_name }}</title>
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'selected_course/style.css' %}">
+    <link rel="stylesheet" href="{% static 'navigation_bar/style.css' %}">
 </head>
 <body>
-    <h1>{{ course.course_name }} ({{ course.course_code }})</h1>
-    <h2>Course Sections</h2>
-    <ul>
-        {% for section in sections %}
-            <li>Section {{ section.course_section_number }}: {{ section.start_time }} - {{ section.end_time }}</li>
-        {% empty %}
-            <li>No sections available for this course.</li>
-        {% endfor %}
-    </ul>
+    {% include 'navigation_bar/navigation.html' %}
+    <div class="container">
+        <h1>{{ course.course_name }} ({{ course.course_code }})</h1>
+        <h2>Course Sections</h2>
+        <ul>
+            {% for section in sections %}
+                <li>Section {{ section.course_section_number }}: {{ section.start_time }} - {{ section.end_time }}</li>
+            {% empty %}
+                <li>No sections available for this course.</li>
+            {% endfor %}
+        </ul>
 
-    <h2>Teaching Assistants</h2>
-    <ul>
-        {% for ta in tas %}
-            <li>{{ ta.username }}</li>
-        {% empty %}
-            <li>No teaching assistants assigned to this course.</li>
-        {% endfor %}
-    </ul>
+        <h2>Teaching Assistants</h2>
+        <ul>
+            {% for ta in tas %}
+                <li>{{ ta.username }}</li>
+            {% empty %}
+                <li>No teaching assistants assigned to this course.</li>
+            {% endfor %}
+        </ul>
+
+        <h2>Instructors</h2>
```

Comment:
while we're here, can we add a create section button in the top right for admins and link it to the create section form for this course